strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/repository/NugetRepositoryFeatures.java

```diff
@@ -379,6 +379,10 @@ private WebTarget queryParams(WebTarget path,
             path = path.queryParam("$top", paginator.getLimit());
         }
 
+        //https://chocolatey.org/api/v2/$metadata 
+        // includePrerelease is required parameter to invoke Search() method
+        path = path.queryParam("includePrerelease", nugetSearchRequest.isIncludePreRelease());
```

Comment:
```
Boolean.TRUE.equals(nugetSearchRequest.getIncludePreRelease())
```

Reply:
Just for my knowledge: How does it `Boolean` and `boolean` make difference here. Are we persisting these values in DB.?

Reply:
the difference is that `boolean` can't be `null`, this will be error prone if have it as `boolean` in `NugetSearchRequest`

Reply:
Thanks